3_bidirectional_cxx_interop/cmake/modules/AddSwift.cmake

```diff
@@ -48,6 +48,11 @@ function(_swift_generate_cxx_header target header)
   cmake_path(APPEND base_path ${header}
     OUTPUT_VARIABLE header_path)
 
+  # Work out the name of the .d file that swiftc will emit
+  get_target_property(module_name ${target} Swift_MODULE_NAME)
```

Comment:
`Swift_MODULE_NAME` is an empty string if the property isn't set explicitly. 
We should use `${ARG_MODULE_NAME}` to get the module name, reducing the depfile computation to:

```cmake
  cmake_path(APPEND CMAKE_CURRENT_BINARY_DIR "${ARG_MODULE_NAME}.emit-module.d"
    OUTPUT_VARIABLE depfile_path)
```